### DISCUSSION TITLE
Batan gemi ve filika sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#191] [Zeka Sorusu] [answered]
20 dk içinde batacak bir gemide 15 kişi vardır. Tek kurtulma şansları 5 kişilik bir filikadır. Geminin etrafında da köpek balıkları var yüzüp karaya çıkamıyorlar. Filikanın en yakın adaya gidip gelmesi 9 dk sürüyor. Kaç kişi kurtulur?

**Comment by Berkek007** [comment, marked as answer]
cevap 13 olur.

İlk olarak  5 kişi biner adaya gider. 4'ü orada kalır 1'i filikayı geri getirir (filikanın tesla olmadığını varsayıyoruz). O da 4 kişi daha alır ve filika 5 kişi olur. Adaya tekrar 4 kişi bırakılır biri döner teknenin batmasına 2 dk kalır. Ama daha tekne batmadığı için filikayla bir 4 kişi daha alırız ve tekne bizden sonra batar. Son olarak sürücü dahil 5 kişi iner ve adaya köpek balıklarına yem olan iki kişi hariç herkes yani 5 + 4 + 4 = 13 kişi sağ bir şekilde ulaşır.
